Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -1459,15 +1500,12 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
             delete this.capiMock;
         },
 
-        "Should reset the user and provide the response when logging out": function () {
-            var logOutResponse = {},
-                logOutError = {};
-
+        _mockExpectations: function () {
             Y.Mock.expect(this.capiMock, {
                 method: 'logOut',
                 args: [Y.Mock.Value.Function],
                 run: function (cb) {
-                    cb(logOutError, logOutResponse);
+                    cb(this.logOutError, this.logOutResponse);
```

Comment:
`this` in a `run` function is the mock being configured. So here `this.logOutError` and `this.logOutResponse` are actually `undefined`.